packages/formatron/src/dataTypes/related.ts

```diff
@@ -13,9 +13,9 @@ export abstract class RelatedType extends DataType {
     return this.field.options?.relatedTo;
   }
 
-  get firstRelationTemplate() {
+  getFirstRelationTemplate() {
     const [firstId] = this.templateIds ?? [];
-    return firstId ? this.formatron.getTemplateById(firstId) : undefined;
+    return firstId ? this.formatron.getModelTemplate({id: firstId}) : undefined;
```

Comment:
This data type requires 1+ related template IDs so we should throw / return an error if it doesn't contain one.